newIDE/app/src/InstancesEditor/InstancesRenderer/LayerRenderer.js

```diff
@@ -221,13 +311,12 @@ export default class LayerRenderer {
    */
   _isInstanceVisible(instance: gdInitialInstance) {
     //TODO: Properly handle rotation
```

Comment:
I believe this TODO can be removed as you fixed the culling thanks to the proper AABB when objects are rotated :)

Reply:
Ok, I wasn't sure, I'll remove it.